internal/tracer.go

```diff
@@ -74,7 +74,6 @@ func (t *tracingContextPropagator) Inject(
 	// retrieve span from context object
 	span := opentracing.SpanFromContext(ctx)
 	if span == nil {
-		t.logger.Info("could not retrieve span from context")
```

Comment:
Why we don't need these logging information?

Reply:
These logs are noisy. I added them to see how much they will spew.. But if someone enables a tracer but doesnt start traces - which will be common in the start workflow scenario this will flood logs..

Reply:
Makes sense. Thanks for the explanation.